src/notifications/github_commenter.rs

```diff
@@ -85,6 +86,36 @@ impl GithubCommenter {
             .await?)
     }
 
+    /// Posts a comment to issue `issue_number` on GitHub repo `repo` with owner `owner`, containing a
+    /// message notifying that a PR comparison has been started and including the run info from
+    /// `base_run` and `head_run`
+    /// Returns an error if creating the message or posting it to GitHub fails
+    pub async fn post_pr_run_started_comment(
+        &self,
+        owner: &str,
+        repo: &str,
+        issue_number: i32,
+        base_run: &RunData,
+        head_run: &RunData,
+        test_name: &str,
+    ) -> Result<(), Error> {
+        let base_run_as_string = serde_json::to_string_pretty(base_run)?;
+        let head_run_as_string = serde_json::to_string_pretty(head_run)?;
+        let comment_body = format!(
+            "### 🥕CARROT🥕 PR comparison started\n\
```

Comment:
Nice.  I never thought about adding unicode to source code files.

Reply:
Pretty neat, right?  Time to start replacing all my variable and function names with emoji

Reply:
🍗 = 🔥(🦆);
😊 = 🍽(🍗);

hmm… This might be a bridge too far…